pdfminer/cmapdb.py

```diff
@@ -361,8 +361,8 @@ def do_keyword(self, pos, token):
         if token is self.KEYWORD_ENDCIDCHAR:
             objs = [obj for (__, obj) in self.popall()]
             for (cid, code) in choplist(2, objs):
-                if isinstance(code, str) and isinstance(cid, str):
-                    self.cmap.add_code2cid(code, nunpack(cid))
+                if isinstance(code, bytes) and isinstance(cid, int):
+                    self.cmap.add_cid2unichr(cid, code)
```

Comment:
I'm not familiar with this part of the code. 

Are you positively certain that this is how it should be?

Reply:
It works in my situation and do not break any case. My concern is that a font with cmap type 13/14 which cid and code are  not one-to-one mapping may not work, but I can find this type of pdf file.